x/service/module/autocli.go

```diff
@@ -0,0 +1,51 @@
+package service
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		// TODO_TECHDEBT(#370): Integrate with AutoCLI
```

Comment:
Commenting this out is not necessary and will confuse ignite upon future message scaffolding in this module.